eng/source-build/source-build.proj

```diff
@@ -0,0 +1,52 @@
+<Project Sdk="Microsoft.DotNet.Arcade.Sdk" DefaultTargets="Build">
+
+  <Import Project="../SourceBuild.props" />
+
+  <PropertyGroup>
+    <ProjectRoot>$(MSBuildThisFileDirectory)../../</ProjectRoot>
+  </PropertyGroup>
+
+  <Target Name="Build" DependsOnTargets="SourceBuildPostBuild">
+    <Message Text="Running source-build Build target" Importance="High" />
+  </Target>
+
+  <Target Name="SourceBuildPreBuild">
```

Comment:
What's the purpose of this?  Is this an extension point just in case it is needed?

Reply:
Yes, this is where the restore and feed setup for intermediate source-build packages will go.

Reply:
Makes sense, thanks.